Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -356,6 +360,13 @@ open class TextStorage: NSTextStorage {
 
             processUnderlineDifferences(in: domRange, betweenOriginal: sourceStyle, andNew: targetStyle)
         case NSAttachmentAttributeName:
+            if sourceValue is LineAttachment || targetValue is LineAttachment {
```

Comment:
@diegoreymendez I added a test and it looks it's working ok, do you mind to have a look on it?